ams/optimizer/spark-optimizer/src/main/java/com/netease/arctic/optimizer/spark/SparkFunction.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.optimizer.spark;
+
+import com.netease.arctic.ams.api.OptimizingTask;
+import com.netease.arctic.ams.api.OptimizingTaskResult;
+import com.netease.arctic.optimizing.OptimizingExecutor;
+import com.netease.arctic.optimizing.OptimizingExecutorFactory;
+import com.netease.arctic.optimizing.OptimizingInputProperties;
+import com.netease.arctic.optimizing.TableOptimizing;
+import com.netease.arctic.utils.ExceptionUtil;
+import com.netease.arctic.utils.SerializationUtil;
+import org.apache.iceberg.common.DynConstructors;
+import org.apache.spark.api.java.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.ByteBuffer;
+
+public class SparkFunction implements Function<OptimizingTask, OptimizingTaskResult> {
```

Comment:
Class docs is required, and I suggest naming the class "SparkOptimizingTaskExecuteFunction," as it is clearer.